packages/stream-metadata/src/riverStreamRpcClient.ts

```diff
@@ -206,12 +207,13 @@ export async function getMediaStreamContent(
 
 	const secretHex = toHexString(secret)
 	const ivHex = toHexString(iv)
+	const cacheKey = `${fullStreamId}${secretHex}${ivHex}`
```

Comment:
I'd suggest making this a hash of these, so you can also have a shorter value to use as a key to store this in redis in the future by the same key.

Reply:
And move it into the if block, so if caching isn't enabled, it isn't computed